### DISCUSSION TITLE
Lap Time Comparison

### DISCUSSION THREAD

**Discussion by TheLastSilva** [snipem/gt7dashboard#27] [Ideas]
The program is working great for me.
Observing the operation of the program, I understood that I can only compare the last lap with any other lap I have done. Which has helped me a lot to improve my times.
So I'll give you an idea here, would it be possible to choose the two turns that I choose to compare? (not limited to just the last lap which seems to be fixed).

**Comment by snipem** [comment]
You mean that one can select the blue lap like one can select the magenta lap?

**Reply by TheLastSilva** [reply]
Yes, that's right. The way I use this dashboard, I end up doing 20 laps, and sometimes the laps I want to compare aren't exactly the last lap or the fastest.

**Reply by snipem** [reply]
I have that on my list. For now the only way to achieve this is to save the laps, delete the laps you do not want to see in a text editor and load the laps again.

**Reply by TheLastSilva** [reply]
Could it be Notepad for example?

**Reply by snipem** [reply]
Any editor that opens json files will do. Notepad for example.
